pkg/gen/filters/common/helper.go

```diff
@@ -45,3 +47,24 @@ func Pluralize(s string, i int) string {
 	}
 	return plural.Plural(s)
 }
+
+func MakeListOfFields[T any](inputList []T, fieldName string) ([]string, error) {
```

Comment:
That's a funny one. Instead of generics maybe you just use a []any as inputList. Reflect anyway will just use any.